module-2/backend/week_three.md

```diff
@@ -10,14 +10,69 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+
+ `rails new project_name -T --database=postgresql --skip-turbolinks --skip-spring`
+
 2. What do Models generally inherit from in rails?
+
+ `< ApplicationRecord`
+
 3. What do Controllers generally inherit from in a rails project?
+
+ `< ApplicationController`
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+
+ `get '/show', to: 'articles#show'`
+ 
 5. What rake task is useful when looking at routes, and what information does it give you?
+
+ `rake routes`
+ It lists all the routes in the controller with their prefix, verb, URI pattern, and controller method.
+
 6. What is an example of a route helper? When would you use them?
+
+ An example is `article_path(article`. We se them to specify a route outside of the router, like in an erb file.
+
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+
+ `_url` gives a path from the root (including whole url) and `_path` gives a relative path.
```

Comment:
Right idea. I wouldn't say that `_url` is from the root since it does include the whole URL.